tests/testthat/test-operators-replace-na.txt

```diff
@@ -0,0 +1,33 @@
+
+%|% fails on non-atomic original values
+=======================================
+
+> call("fn") %|% 1
+Error: Cannot replace missing values in an object of type language
+
+
+%|% fails with wrong types
+==========================
+
+> c(1L, NA) %|% 2
+Error: The replacement values, which have type double, must have the same type as the original values, which have type integer
```

Comment:
These messages are rather long (~120 characters), ideally they'd fit within 72 or 80 characters.

Maybe "Replacement values must have type foo, not type bar".